app/models/notification.rb

```diff
@@ -96,32 +96,34 @@ def update_from_api_response(api_response, unarchive: false)
 
   def download_subject
     user.github_client.get(subject_url)
+  rescue Octokit::Forbidden, Octokit::NotFound
```

Comment:
Hmm is there some way we can flag this so it doesn't get buried when it's what we expected should work? An entry in the logs, perhaps?